index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/"
+        crossorigin="anonymous">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="main-content">
+        <div class="topbar">
+            <button type="button" class="topbar-button"><i class="fab fa-apple"></i></button>
+            <button type="button" class="topbar-button">Store</button>
+            <button type="button" class="topbar-button">Mac</button>
+            <button type="button" class="topbar-button">iPad</button>
+            <button type="button" class="topbar-button">iPhone</button>
+            <button type="button" class="topbar-button">iTunes</button>
+            <button type="button" class="topbar-button">Support</button>
+            <form action="#" class="topbar-search">
+                <input type="search" placeholder="Search">
+            </form>
+        </div>
+        <div class="header">
+            <h1>What will your verse be?</h1>
+            <h2>Everyone has something to share. See how people are using iPad in their unique ways to add to the
+                world’s story.</h2>
+            <h3>Watch the “Your Verse” film and explore the stories</h3>
+        </div>
+    </div>
+    <div class="footer-grid">
+        <div class="footer-element">
+            <div><strong style="font-size: 22px;">iPhone 5[S]</strong></div>
+            <br>
+            <div>Forward thinking.</div>
+        </div>
+        <div class="footer-element">
+            <div><strong style="font-size: 22px;">iPhone 5[C]</strong></div>
+            <br>
+            <div>For the colorful.</div>
+        </div>
+        <div class="footer-element">
+            <div><strong style="font-size: 22px;">Happy Birthday, Mac.
+                </strong></div>
+            <br>
+            <div>Explore years of innovation on the Mac.
+            </div>
+        </div>
+        <div class="footer-element">
+            <div><strong style="font-size: 22px;">Supplier Responsibility</strong></div>
+            <br>
+            <div>The Latest result of our worldwide effors
+            </div>
+        </div>
+    </div>
+    <div class="footer">
+            Shop the Apple Online Store (1-800-MY-APPLE), visit an Apple Retail Store, or find a reseller.Site MapHot NewsRSS FeedsMedia InfoEnvironmentJob OpportunitiesContact UsChoose your country or region Copyright © 2014 Apple Inc. All rights reserved. Terms of UsePrivacy Policy
```

Comment:
This seems just like a copy paste. The original has links and a layout for them. Try to be as close to the original as you can